auth/conf.py

```diff
@@ -5,15 +5,28 @@
 
 
 # database related configuration
-dbName = os.environ.get("DB_NAME", "postgres")
-dbUser = os.environ.get("DB_USER", "auth")
-dbPdw = os.environ.get("DB_PWD", "")
-dbHost = os.environ.get("DB_HOST", "postgres")
+dbName = os.environ.get("AUTH_DB_NAME", "postgres")
```

Comment:
Now that Auth is growing,  there is a lot of configuration available. 
I think its is a bit messy to configure all these stuff by environments variables. Maybe its time to create a configuration file?

Reply:
Yes, having that option sounds like a good idea.
However, I'd advocate to keeping things configurable through environment variables as well (seems to be a kind of 'de facto' standard for docker-deployed services.